clusterloader2/testing/l4lb/config.yaml

```diff
@@ -6,8 +6,8 @@
 {{$LOAD_BALANCER_TYPE := DefaultParam .CL2_LOAD_BALANCER_TYPE "EXTERNAL"}}
 # $EXTERNAL_TRAFFIC_POLICY specifies the externalTrafficPolicy on LB type service. Valid values are "Cluster" and "Local"
 {{$EXTERNAL_TRAFFIC_POLICY := DefaultParam .CL2_EXTERNAL_TRAFFIC_POLICY "Cluster"}}
-# $NODE_SYNC_TIMEOUT specifies the timeout to wait for nodesync to complete
-{{$NODE_SYNC_TIMEOUT := DefaultParam .CL2_NODE_SYNC_TIMEOUT "30m"}}
+# $SYNC_TIMEOUT specifies the timeout to wait for nodesync and LB creation to complete
+{{$SYNC_TIMEOUT := DefaultParam .CL2_SYNC_TIMEOUT "30m"}}
```

Comment:
Thinking about a future - it would be good to have distinct names, as multiple configs could be run in a single job.

Maybe `CL2_LB_SYNC_TIMEOUT` or `CL2_L4LB_SYNC_TIMEOUT` ?

Reply:
Done. Thanks